lib/lookip.js

```diff
@@ -38,9 +46,14 @@ exports.mask = (cleanIpString) => {
   } else if (ip.isV6Format(cleanIpString)) {
     return ip.mask(cleanIpString, 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:0');
   } else {
-    return null;
+    return cleanIpString;
   }
 }
+exports.maskLeft = (cleanXffString) => {
```

Comment:
So why mask only the left most, and not each IP? b/c only the left most should be the user's individual IP, and we are fine keeping full IPs for the proxies or other machines in there after that? I think that makes sense.

Reply:
Correct, but background...

In BigQuery, we _only_ store the masked left-most IP of the XFF header (the originating user's IP).  And we just discard any proxies that happened on the way.  Maybe we should be keeping those?  But currently we're not.

In pingbacks, we were previously just passing on the XFF header in the GET request.  I decided to maintain that, but mask the left-most user IP string.  Assuming that the other IPs in there (pretty much just our ALB) aren't identifying info.